.github/workflows/release.yml

```
@@ -18,32 +18,40 @@ jobs:
+            -d '{"tag_name": "preview", "target_commitish": "${{ steps.release_action.outputs.commit_short_sha }}"}' \
+            https://api.github.com/repos/${{ github.repository }}/git/refs/tags/preview
+
+      # Update main tag if not a prerelease
+      - name: Update main tag
         if: github.event.release.prerelease == false
-
-      - name: Actions for Discord (Preview)
-        uses: Ilshidur/action-discord@d2594079a10f1d6739ee50a2471f0ca57418b554
-        env:
-          DISCORD_WEBHOOK: ${{ secrets.TAILSCALE_DISCORD_WEBHOOK }}
+        run: |
```

actionlint (1.7.7)

property "release_action" is not defined in object type {}

(expression)